rules/aip0123/resource_type_name.go

```diff
@@ -15,29 +15,53 @@
 package aip0123
 
 import (
+	"fmt"
+	"regexp"
 	"strings"
+	"unicode"
 
 	"github.com/googleapis/api-linter/lint"
 	"github.com/googleapis/api-linter/locations"
 	"github.com/googleapis/api-linter/rules/internal/utils"
 	"github.com/jhump/protoreflect/desc"
 )
 
+var (
+	resourceNameRegex = regexp.MustCompile("^[\\p{L}A-Za-z0-9]+$")
```

Comment:
```suggestion
	resourceNameRegex = regexp.MustCompile(`^[\\p{L}A-Za-z0-9]+$`)
```

This makes the go linter happy

Reply:
I see several other instances of using `"` in stead of backtick...likely before the golang lint was added 🤷🏻

Reply:
thank you! I just saw that as well. fixed.